pkg/apitype/policy.go

```diff
@@ -107,3 +107,27 @@ type GetStackPolicyPacksResponse struct {
 	// RequiredPolicies is a list of required Policy Packs to run during the update.
 	RequiredPolicies []RequiredPolicy `json:"requiredPolicies,omitempty"`
 }
+
+// UpdatePolicyGroupRequest modifies a Policy Group.
+type UpdatePolicyGroupRequest struct {
+	NewName *string `json:"newName,omitempty"`
+
+	AddStack    *PulumiStackReference `json:"addStack,omitempty"`
+	RemoveStack *PulumiStackReference `json:"removeStack,omitempty"`
+
+	AddPolicyPack    *PolicyPackMetadata `json:"addPolicyPack,omitempty"`
+	RemovePolicyPack *PolicyPackMetadata `json:"removePolicyPack,omitempty"`
+}
+
+// PulumiStackReference contains a name and some information for the frontend to construct a route.
```

Comment:
"some information for the frontend to construct a route" doesn't make sense when the code is part of the Pulumi CLI.

Perhaps a more appropriate name would be "Qualified Stack Name" with just "StackName" and "ProjectName"? (Since the "OwnerName" is implied.)